src/utils/formHelperFunctions.ts

```diff
@@ -86,7 +86,11 @@ const getAllElements = (step: any) => {
 const lookUpTrigger = (step: any, elementID: any, elementType: any) => {
   if (elementType === 'button') {
     const element = step.buttons.find((button: any) => button.id === elementID);
-    return { id: elementID, text: element.properties.text };
+    return {
+      id: elementID,
+      text: element.properties.text,
+      link: element.properties.link
```

Comment:
why is link being returned here?

Reply:
needed to know the link type for something before, but its no longer needed - removed